babel.config.js

```diff
@@ -0,0 +1,38 @@
+const commonConfig = {
+  presets: [["@babel/env", { loose: true }], "@babel/react"],
+  plugins: ["dev-expression"]
+};
+
+module.exports = {
+  babelrcRoots: [".", "packages/react-router-native/"],
+  overrides: [
+    {
+      test: "./packages/react-router/modules/*",
+      presets: commonConfig.presets,
+      plugins: commonConfig.plugins.concat([
+        ["@babel/proposal-class-properties", { loose: true }]
+      ])
+    },
+    {
+      test: "./packages/react-router-dom/modules/*",
+      presets: commonConfig.presets,
+      plugins: commonConfig.plugins.concat([
+        ["@babel/proposal-class-properties", { loose: true }]
+      ])
+    },
+    {
+      test: "./packages/react-router-config/modules/*",
+      presets: commonConfig.presets,
+      plugins: commonConfig.plugins
+    },
+    {
+      test: "./packages/website/modules/*",
```

Comment:
We can leave the Babel config for the website in the website dir. We don't want to include it in the tests anyway.